SFTPClientCLI/conanfile.py

```diff
@@ -0,0 +1,13 @@
+from conans import ConanFile
+
+class SFTPClientCLIConan(ConanFile):
+	name = "SFTPClientCLI"
+	description = "CLI component that uses SFTP library"
+	author = "CSW <csw@werfen.com>"
+	topics = ("conan", "sftp")
+	license = "MIT"
+	generators = "visual_studio"
+	settings = "os", "compiler", "build_type", "arch"
+
+	def requirements(self):
+		self.requires("libssh/0.9.5@systelab/stable")
```

Comment:
Don't we require the openssl binaries too? 
And maybe zlib, but I am not sure right now if we are linking with it statically or dynamically.

Reply:
And still more important than that, we are not copying the dll to the target folder!

Reply:
For the Openssl, after investigating it properly it seems not. We have built the libssh statically with openssl, thus it is a build depedency of that packet and not a regular dependency (not visible for clients of libssh).
I have adjusted the packet definition of libssh to consider this and thus openssl will not appear anywhere on the sftp adapter projects.